packages/cubejs-dremio-driver/README.md

```diff
@@ -9,11 +9,24 @@
 
 Pure Javascript Dremio driver.
 
+## Dremio Cloud
+
+To use this driver with [Dremio Cloud](https://docs.dremio.com/cloud/reference/api/), use the following setup:
+
+| Environment Variable | Value                                              |
+| -------------------- | -------------------------------------------------- |
+| CUBEJS_DB_TYPE       | dremio                                             |
+| CUBEJS_DB_URL        | https://api.dremio.cloud/v0/projects/${PROJECT_ID} |
+| CUBEJS_DB_NAME       | ${DB_NAME}                                         |
+| CUBEJS_DB_PASS       | Bearer ${PERSONAL_ACCESS_TOKEN}                    |
+
+> It's important to note that "Bearer" is required for using personal access tokens.
```

Comment:
Having a special env just for the token itself also will make it more user-friendly and convenient for usage: just copy/paste the token without need to remember to add `Bearer ` prefix. You can add it in the code because you know exact flow when it's needed